src/views/Dash.tsx

```diff
@@ -38,13 +39,34 @@ export default function Dash() {
     setProjectDetails(details);
   }
 
+  async function validityCheck(isValid: boolean) {
+    if (isValid) {
+      getProjectDetails();
+    } 
+    else {
+      await window.contract.delete_project({
```

Comment:
Don't use `.catch` wrap it in a try-catch and handle the error in the catch section